src/pages/guides/concepts/migration.md

```diff
@@ -0,0 +1,420 @@
+  /**
+ * If target app doesn't open within this time (in ms, same as of setTimeout),
+ * the error callback is invoked with error code TARGET_LOAD_TIMED_OUT.
+ */
+  loadTimeout?: number;
+}
+```
+
+### Full editor API parameters
+
+The modal params are the same for all editor-specific APIs. We have the inheritance of input and output params, as shown below. 
+
+#### Editor App Params inherit Base App params
+
+```ts
+{
+  /** Category to show by default */
+  defaultCategory?: EditorPanelView;
+  /** Search text to pass in the editor for selected panel. */
+  categorySearchText?: string;
+  /** Canvas template type */
+  templateType?: TemplateType;
+  /** Properties to configure the Editor and Modal titles */
+  editorTitle?: string;
+  publishModalTitle?: string;
+  publishErrorModalTitle?: string;
+}
+```
+
+#### Editor Export Params inherit Base Export params
+
+```ts
+{
+  exportOptions?: ExportOptionsEditor[];
+  /**
+ * Decides whether the multiple pages can be exported
+ * @default true
+ */
+  multiPage?: boolean;
+  /**
+ * Specify the list of file types that the user can download.
+ * This can be used to limit the download options as per file types for end users.
+ * This limitation is applied to both native download and custom download scenarios.
+ */
+  allowedFileTypes?: FileType[];
+  /**
+ * Value between 0 and 1 to control the quality of the image.
+ * Currently, only supported for ImageFileType.JPEG when exporting.
+ * @minimum 0
+ * @maximum 1
+ */
+  imageQuality?: number;
+}
+```
+
+### Full Editor API-Specific Input Params
+
+#### Create Doc Params inherits Editor Doc Params
+
+```ts
+{
+  /** Canvas and template size or aspect ratio. */
+  canvasSize?: Size | CanvasAspectId;
+}
+```
+
+#### Create with Asset Doc params inherits Create Doc params 
+
+```ts
+{
+  /** Asset from the client application to start editing with. */
+  asset: AssetInfo;
+}
+
+```
+
+#### Create with Template Doc params inherits Editor Doc params
+
+```ts
+{
+  /** Start the editor workflow with a template */
+  templateId: string;
+}
+```
+
+#### Edit Doc Params inherit Editor Doc Params
+
+```ts
+{
+  /** Document Id of the Express project */
+  documentId: string;
+}
+```
+
+### Module API parameters
+
+We have the same Modal Params as the base params for Modules.
+
+### Module API-Specific Input Params
+
+#### Image Module
+
+**Image Module Doc Params inherits base doc params**
+
+```ts
+{
+  /** Asset from the host application to start editing with. */
+  asset: Asset;
+}
+
+```
+**Image Module Export Params inherits base Export params**
+
+```ts
+{
+  /** Export options to be displayed in Image Module. */
+  exportOptions?: ExportOptionsEditor[];
+  /**
+ * Specify the list of file types that the user can download.
+ * This can be used to limit the download options as per file types for end users.
+ * This limitation is applied to both native download and custom download scenarios.
+ */
+  allowedFileTypes?: FileType[];
+}
+```
```

Comment:
```ts
{
  /** Export options to be displayed in Image Module. */
  exportOptions?: ExportOptionsEditor[];
  /**
   * Specify the list of file types that the user can download.
   * This can be used to limit the download options for end users 
   * based on file types. This limitation is applied to both native 
   * download and custom download scenarios.
   */
  allowedFileTypes?: FileType[];
}
```